go/billing/templatetags/billing_tags.py

```diff
@@ -3,32 +3,42 @@
 from django import template
 from django.utils.translation import ungettext
 
+from go.base.utils import format_currency
 from go.billing import settings
 
 register = template.Library()
 
 
+@register.filter
+def format_cents(v):
+    """returns a formatted string representation of a number in dollars from a
+    decimal cents value (cents are the billing system's internal representation
+    for monetary amounts).
+    """
+    v = v / Decimal('100.0')
```

Comment:
At the moment we are assuming any cents value given is a decimal. This fits our purposes at the moment. The behaviour we are handling at the moment for `format_credits` is actually wrong, we should also always expect decimals, changing it do that.